draft-ietf-quic-recovery.md

```diff
@@ -353,14 +353,17 @@ endpoint:
 On the first RTT sample for a network path, the smoothed_rtt is set to the
 latest_rtt.
 
-smoothed_rtt and rttvar are computed as follows, similar to {{?RFC6298}}.  On
-the first RTT sample for a network path:
+smoothed_rtt and rttvar are computed as follows, similar to {{?RFC6298}}.  When
+there is less than one sample for a network path:
 
 ~~~
-smoothed_rtt = latest_rtt
-rttvar = latest_rtt / 2
+smoothed_rtt = rtt_sample
+rttvar = rtt_sample / 2
 ~~~
 
+where rtt_sample is either the initial RTT when there is no sample, or the
+latest RTT when there is one.
+
 On subsequent RTT samples, smoothed_rtt and rttvar evolve as follows:
```

Comment:
I'm concerned this section is now potentially ambiguous.  I believe we want to maintain the property that the first actual RTT measurement replaces the initialized values, and I don't think that's clear enough here.